stablehlo/tests/interpret_slice.mlir

```diff
@@ -0,0 +1,51 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: slice_op
+func.func @slice_op() -> tensor<2x2xi64> {
+  %operand = stablehlo.constant dense<[[0, 0, 0, 0], [0, 0, 1, 1], [0, 0, 1, 1]]> : tensor<3x4xi64>
+  %result = "stablehlo.slice"(%operand) {
+    start_indices = dense<[1, 2]> : tensor<2xi64>,
+    limit_indices = dense<[3, 4]> : tensor<2xi64>,
+    strides = dense<1> : tensor<2xi64>
+  } : (tensor<3x4xi64>) -> tensor<2x2xi64>
+  func.return %result : tensor<2x2xi64>
+  // CHECK-NEXT: tensor<2x2xi64>
+  // CHECK-NEXT: 1 : i64
+  // CHECK-NEXT: 1 : i64
+  // CHECK-NEXT: 1 : i64
+  // CHECK-NEXT: 1 : i64
```

Comment:
Would it be better to have the results in the following form `[[1, 1], [1, 1]] : tensor<2x2xi64>`
It would require only one CHECK-NEXT line and make it easier to write and understand the results.
For tests with more dimensions it is harder to understand what the result is, we can still have a separate line for type.